auditbeat/docs/breaking.asciidoc

```diff
@@ -0,0 +1,129 @@
+[[auditbeat-breaking-changes]]
+== Breaking changes in 6.2
+
+As a general rule, we strive to keep backwards compatibility between minor
+versions (e.g.  6.x to 6.y) so you can upgrade without any configuration file
+changes, but there are breaking changes between the earlier beta releases and
+the 6.2 GA release.
+
+There are changes that affect both the configuration and the event schema.
+
+[float]
+=== Configuration Changes
+
+The modules and metricsets were renamed and your configuration must be updated.
+
+The `kernel` metricset has been renamed to the
+<<auditbeat-module-auditd,auditd module>>.
+
+.Old Config
+[source,yaml]
+----
+- module: audit
+  metricsets: ["kernel"]
+  kernel.resolve_ids: true
+  kernel.failure_mode: silent
+  kernel.backlog_limit: 8196
+  kernel.rate_limit: 0
+  kernel.include_raw_message: false
+  kernel.include_warnings: false
+  kernel.audit_rules: |
+    # Rules
+----
+
+.New Config
+[source,yaml]
+----
+- module: auditd
+  resolve_ids: true
+  failure_mode: silent
+  backlog_limit: 8196
+  rate_limit: 0
+  include_raw_message: false
+  include_warnings: false
+  audit_rules: |
+    # Rules
+----
+
+The `file` metricset has been renamed to the
+<<auditbeat-module-file_integrity,file_integrity module>>.
+
+.Old Config
+[source,yaml]
+----
+- module: audit
+  metricsets: [file]
+  file.paths:
+  - /bin
+  - /usr/bin
+  - /sbin
+  - /usr/sbin
+  - /etc
+  file.scan_at_start: true
+  file.scan_rate_per_sec: 50 MiB
+  file.max_file_size: 100 MiB
+  file.hash_types: [sha1]
+----
+
+.New Config
+[source,yaml]
+----
+- module: file_integrity
+  paths:
+  - /bin
+  - /usr/bin
+  - /sbin
+  - /usr/sbin
+  - /etc
+  scan_at_start: true
+  scan_rate_per_sec: 50 MiB
+  max_file_size: 100 MiB
+  hash_types: [sha1]
+  recursive: false <1>
+----
+<1> `recursive` is a new option in 6.2 and is disabled by default. By setting
+the value to `true` it will watch for changes in all sub-directories.
+
+[float]
+=== Event Schema Changes
+
+Most field names were changed in 6.2. This is because of the module renames as
+well as a desire to use common field names for similar data types between
+the modules (and future modules). The table below provides a summary of the
+field changes.
+
+If you are using the default index naming and template settings then no action
+is required because the Auditbeat version number is included in then index
+and template names. This prevents any field mapping issues between versions.
+
+In Kibana you will need to import the latest dashboards that work with the
```

Comment:
Just say, "In Kibana you need to import....

Reply:
might be worth providing the command here for importing the dashboards? Or a link to the docs?